### PR TITLE
Feature/contract agreement power of attorney

### PR DESCRIPTION
**Pull request by meletios**
_First check https://github.com/HedvigInsurance/product-pricing/pull/373 (Contract blacklist)_

Add power of attorney bytearray to contracts

### CODE REVIEW

File: src/main/java/com/hedvig/productPricing/service/events/InsuranceMandateCreatedEvent.java [hunk 1]
```diff
@@ -1,14 +1,43 @@
 package com.hedvig.productPricing.service.events;
 
-import lombok.Value;
+public final class InsuranceMandateCreatedEvent {
+  private final String memberId;
+  private final byte[] mandatePdf;
```
Comment (vonElfvin) [inline src/main/java/com/hedvig/productPricing/service/events/InsuranceMandateCreatedEvent.java]: You could just make these public so you don't need all the "delombok" code below

File: src/main/java/com/hedvig/contract/util/Country.kt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package com.hedvig.contract.util
+
+enum class Country {
+  SWEDEN
+}
```
Comment (vonElfvin) [inline src/main/java/com/hedvig/contract/util/Country.kt]: I think we should use a package for this so we can seemlessly handle multiple types of country representations (country code etc.)  I did some digging before and this one seems like the best: https://github.com/TakahikoKawasaki/nv-i18n

**Comment by meletios** [comment]
> We may need to discuss this with Fredrik and Nordlin, implementationwise it looks good though

Correct, i will not merge this until i have the discussions with Fredrik and Karl

**Comment by meletios** [comment]
Will be stored in the member-service.

**Pull request closed without merging**